tests/ui/traits/const-traits/const-fns-are-early-bound.rs

```diff
@@ -1,7 +1,4 @@
-//@ known-bug: #110395
```

Comment:
what is this testing? it reimplements mini-core :/ i'd like to potentially remove this test and integrate it in `minicore-works.rs`